pkg/agent/agent.go

```diff
@@ -222,6 +225,31 @@ func (i *Initializer) Initialize() error {
 		return err
 	}
 
+	// Periodically check whether IP configuration of the gateway is correct.
+	// Terminated when stopCh is closed.
+	go wait.Until(func() {
```

Comment:
I think networkpolicyonly mode shouldn't call this, why don't just start a goroutine calling `util.ConfigureLinkAddresses` after it's first called?